### DISCUSSION TITLE
Is it possible to extend Toolbars click area on Taskbar?

### DISCUSSION THREAD

**Discussion by doterik** [valinet/ExplorerPatcher#750] [Q&A] [answered]
Many thanks for making Windows 11 useful, not least the new weather widget is great.

I use toolbars on the taskbar and wonder if it is at all achievable to expand the area to click on. As it is now, you have to aim quite well to hit the toolbar. Would it be **_possible_** to make it so that you can also click on the name?

![image](https://user-images.githubusercontent.com/4037602/151593233-575fb7e9-9108-4dad-8a5c-bbe8f809d554.png)

Keep up the great work and thank you again. 👍🏼

**Comment by valinet** [comment, marked as answer]
Thank you

I don’t really know, I will have to investigate it. If I find something out about it, I will surely mention it.

**Comment by doterik** [comment]
Ok, thanks for your reply.

I guess it's, if not impossible, then at least unnecessarily complicated.

I buy a pair of glasses then I can manage. 😎

###### If you come across something, it would be great...

**Reply by valinet** [reply]
Maybe also have a look at this tool, see if maybe it provides a better alternative for your use case: https://github.com/Hofknecht/SystemTrayMenu

**Reply by doterik** [reply]
Great, it looks promising. I will have a try.

Thanks for your suggestion.

**Reply by valinet** [reply]
You’re welcome. Actually, thank @chip33, he mentioned it the other day and I just happened to remember seeing his post and briefly checking it out.
